operators/tokenizer/clip_tokenizer.cc

```diff
@@ -0,0 +1,627 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
create a bpetokenizer.hpp to put all the common function and classes into it, then remove the code from gpt2_* and clip_* files.